docs/custodian.md

```diff
@@ -0,0 +1,279 @@
+# Custodians
+
+Welcome, and *Thank you* for volunteering to be a custodian of an ArmoredWitness device!
```

Comment:
Is "Thank you" meant to be capitalized?

Reply:
Yeah, I was going for _extra Emphasis!_ :)